packages/side-nav/README.md

```diff
@@ -47,7 +47,7 @@ return (
     <li>
       <h4 class="leafygreen-ui-19w7vet">States</h4>
       <ul role="menu" class="leafygreen-ui-bdnco">
-        <li role="menuitem">
+        <li role="none">
           <button
```

Comment:
Shouldn't the buttons then have role="menuitem" ? 
See HTML source doe here: https://www.w3.org/TR/wai-aria-practices-1.1/examples/menu-button/menu-button-links.html

Reply:
Oops, @DesignerDave good catch. From W3's spec, an example:
```
<li role="none">
  <a role="menuitem" href="https://www.w3.org/standards/webdesign/accessibility">
    W3C Web Accessibility Initiative
  </a>
</li>
```
So by default if `<ul role="menu">` is your container, `li` elements treated as menu items since they're direct descendants. Since the actual interactive element is the link/button/whatever, we have to reset the role back to `role="none"`.

Does that sound right?

Reply:
Ahh, yep! Sorry, I believe this is implemented correctly, just forgot to add the role in the README.

Reply:
Shouldn't the button have role="menuitem"?

Reply:
I only see the role set on the LI

Reply:
I'm seeing it as an attribute of the button in the PR. It was already there prior, so it's not showing up as a change.

Reply:
got it.. the compare view was cutting a bit out so I didn't notice